framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -191,6 +227,82 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 		await setChainConnectorInfo(this._chainConnectorPluginDB, chainConnectorInfo);
 	}
 
+	public async calculateCCUParams(
```

Comment:
is this mentioned in any LIP as appendix?

Reply:
no its not mentioned anywhere how to calculate CCU Params, just overview of the process to create CCU here https://github.com/LiskHQ/lips/blob/main/proposals/lip-0053.md#creating-and-posting-cross-chain-update-transactions